packages/wrangler/src/d1/export.ts

```diff
@@ -101,26 +114,77 @@ async function exportRemotely(
 
 	logger.log(`🌀 Executing on remote database ${name} (${db.uuid}):`);
 	logger.log(`🌀 Creating export...`);
-	const metadata = await fetchResult<ExportMetadata>(
-		`/accounts/${accountId}/d1/database/${db.uuid}/export`,
-		{
-			method: "POST",
-			headers: {
-				"Content-Type": "application/json",
-			},
-			body: JSON.stringify({
-				outputFormat: "file",
-				dumpOptions: {
-					noSchema,
-					noData,
-					tables,
-				},
-			}),
-		}
-	);
+	const dumpOptions = {
+		noSchema,
+		noData,
+		tables,
+	};
+
+	const finalResponse = await pollExport(accountId, db, dumpOptions, undefined);
 
-	logger.log(`🌀 Downloading SQL to ${output}`);
-	const contents = await fetch(metadata.signedUrl);
+	if (finalResponse.status !== "complete")
+		throw new Error(`Error: D1 reset before export completed!`);
```

Comment:
```suggestion
		throw new Error(`D1 reset before export completed!`);
```
Throwing an error already adds "error" to the console.

Also is this an APIError or a UserError?

Reply:
API Error